torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -43,10 +40,15 @@
 }
 
 
-# Uses PTD FSDP AC wrapper
-# currently selective per op and per layer checkpointing are supported
-def checkpoint_wrapper(module, config):
-    if config.mode == "selective" and config.selective_ac_option == "op":
+def checkpoint_wrapper(module: torch.nn.Module, ac_config):
+    valid_ac_modes = ("full", "selective")
+    if ac_config.mode not in valid_ac_modes:
+        raise ValueError(
```

Comment:
there is one more possibility which should fail: `ac_config.mode == "selective"` but with invalid `selective_ac_option`

Reply:
Good catch! (by the way, I tried to make it like this so that we can include what is valid in the error message)

Reply:
looks good, feel free to merge